app/components/login.jsx

```diff
@@ -1,202 +1,318 @@
-              className         =   "forgot-password-link"
-              onClick           = { this.forgotPassword }
-            >
-            Click here
-            </a>
-          </Column>
-        </Row>
-      </div>
-    );
-
-    if ( this.state.info ) {
-      content = ( <Loading message="Loggin you in..." /> );
-    }
-
-    else if ( this.state.successMessage ) {
-      content = ( <div></div> );
+// https://github.com/EnCiv/civil-pursuit/issues/150
+
+// CHANGELOG
+// 7/11/24: removed instances of useAuth to try to get interface working, will fix later
+// 7/12/24: removed "ref" from login.jsx file since its deprecated, trying to get some stories up
+// 7/13/24: changed file name to login.jsx after installing civil-client
+// 7/15/24: i had to change stuff on webpack-dev to get the email updating properly (buffer is not defined)
+import React, { useState, useEffect } from 'react';
+import cx from 'classnames';
+import { createUseStyles } from 'react-jss';
+import Color from 'color';
+import { useAuth } from  'civil-client';
+
+function Login(props, ref) {
+    
+    const { className, style, onDone = () => {}, startTab = 'login' } = props
+    
+    // checks if start tab requests login or sign up page
+    const [isLogIn, setIsLogIn] = useState(
+        startTab.toLowerCase().includes('up') ? false : startTab.toLowerCase().includes('in') ? true : false
+    )
+    const { destination, userInfo = {} } = props
+    const classes = useStyles()
+    const [state, methods] = useAuth(destination, userInfo)
+
+    // if user has filled out required fields, automatically log them in
+    if (userInfo.email && userInfo.password) {
+        useEffect(() => {
+            onDone({ valid: true, value: userInfo });
+            return
+        })
     }
 
+    // otherwise, continue showing login/sign up page
     return (
-      <Modal className={ Component.classList(this, ...classes) } title="Login">
-        <Form handler={ this.login.bind(this) } flash={ this.state } form-center name="login" ref='form'>
-          { content }
-        </Form>
-      </Modal>
-    );
-  }
+        <div className={cx(className, classes.Login)} style={style} ref={ref}>
+            <div className={classes.tabs}>
+                <div className={cx(classes.tab, !isLogIn && classes.tabSelected)}>
+                    {/* (remove corner decoration) <div className={cx(classes.leftTabCorner, !isLogIn && classes.disabled)}>
+                        <div className={classes.leftTabCornerContent} />
+                    </div> */}
+                    <button onClick={e => setIsLogIn(false)} className={cx(classes.btnClick, !isLogIn && classes.btnClickSelected)}>
+                        Sign Up
+                    </button>
+                </div>
+                <div className={cx(classes.tab, isLogIn && classes.tabSelected)}>
+                    {/* (remove corner decoration) <div className={cx(classes.rightTabCorner, isLogIn && classes.disabled)}>
```

Comment:
Now that you've got it working, please remove commented out code, and such that won't be userfule inf the future.  Or, if something is commented out and there's a reason to keep it for the future include a comment about why.